src/View/Strategy/PdfStrategy.php

```diff
@@ -19,20 +19,15 @@
 
 namespace DOMPDFModule\View\Strategy;
 
+use Zend\View\ViewEvent;
 use DOMPDFModule\View\Model;
 use DOMPDFModule\View\Renderer\PdfRenderer;
 use Zend\EventManager\EventManagerInterface;
-use Zend\EventManager\ListenerAggregateInterface;
-use Zend\View\ViewEvent;
+use Zend\EventManager\AbstractListenerAggregate;
 
-class PdfStrategy implements ListenerAggregateInterface
+class PdfStrategy extends AbstractListenerAggregate
```

Comment:
The AbstractListenerAggregate class already implements the ListenerAggregateInterface interface and it contains the implementation of the detach method in which it was implemented by the ZF community, I do not think it has the need to write the method again knowing that it already exists, I think we should reuse it.